pybliometrics/scopus/abstract_retrieval.py

```diff
@@ -287,6 +287,17 @@ def description(self) -> Optional[str]:
         """
         return chained_get(self._json, ['coredata', 'dc:description'])
 
+    @property
+    def document_entitlement_status(self) -> Optional[str]:
+        """Returns the document entitlement status, i.e. tells if the requestor 
+        is entitled to the requested resource.
+        Note: Only works with `ENTITLED` view.
+        """
+        if self._view == 'ENTITLED':
+            return chained_get(self._json, ['document-entitlement', 'status'])
+        else:
+            return None
```

Comment:
Actually, can we not solve this via try-except? Would be much faster.

Reply:
True... actually the `chained_get` already handles the case where the entitlement status is not found and returns `None`